src/main/scala/scalalab3/chatbotengine/model/Message.scala

```diff
@@ -0,0 +1,11 @@
+package scalalab3.chatbotengine.model
+
+case class Message(val id: Long, val from: User, val date: Integer, val chat: Chat, val forwardFrom: User,
```

Comment:
As I can see not fields are required for creating message. Please use Option here with default value